perllib/FixMyStreet/App/Controller/Offline.pm

```diff
@@ -82,6 +54,72 @@ sub manifest: Path("/.well-known/manifest.webmanifest") {
     $c->res->body($json);
 }
 
+sub _stash_manifest_theme : Private {
+    my ($self, $c, $cobrand) = @_;
+
+    $c->stash->{manifest_theme} = $c->forward('_find_manifest_theme', [ $cobrand ]);
+}
+
+sub _find_manifest_theme : Private {
+    my ($self, $c, $cobrand, $ignore_cache_and_defaults) = @_;
```

Comment:
Now the find/stash thing has been tidied, looking at all the calls, I wonder if it'd be nicer to have the default be ignore (so every direct call to _find_manifest can only pass in the cobrand), and it's only the call in stash_manifest_theme that would have to pass a flag to say "use_cache_and_defaults".

Reply:
I'm 50/50 on this. I agree that the 2 of the 3 calls passing the flag is an indication that it should be the default, but I also think for this specific function the default behaviour should be the most efficient/fastest i.e. to use the cache. Is it compounded by the flag actually doing two separate but unrelated things? Maybe it would be better as a `%params` hash? Dunno.